superset-frontend/cypress-base/cypress/e2e/dashboard_list/list.test.ts

```diff
@@ -195,7 +195,7 @@ describe('Dashboards list', () => {
         .should('not.contain', '4 - Sample dashboard');
     });
 
-    it('should delete correctly', () => {
+    it.skip('should delete correctly', () => {
```

Comment:
@geido provided a fix for this test in https://github.com/apache/superset/pull/31933